<p>Department of Integrative Biology, University of California, Berkeley<br />
University Herbarium, University of California, Berkeley<br />
Center for Population Biology, University of California, Davis<br />
<br />
3040 Valley Life Sciences Building #3140<br />
Berkeley, CA 94720-3140<br />
Email: <a href="mailto:mikeryanmay@gmail.com"><u>mikeryanmay@gmail.com</u></a><br />
Website: <a href="https://mikeryanmay.github.io"><u>mikeryanmay.github.io</u></a></p>

<h2>Education and Appointments</h2>

<table>
  <tbody>
    <tr>
      <td>2020–present</td>
      <td>Postdoctoral scholar, UC Davis</td>
    </tr>
    <tr>
      <td>2019–present</td>
      <td>Postdoctoral scholar, UC Berkeley</td>
    </tr>
    <tr>
      <td>2011–2019</td>
      <td>PhD, Population Biology, UC Davis</td>
    </tr>
    <tr>
      <td>2008–2010</td>
      <td>BS, Evolution, Ecology and Biodiversity, UC Davis (Highest Honors)</td>
    </tr>
  </tbody>
</table>

<h2>Publications</h2>

<ol reversed="reversed">
  <li><strong>May, M. R.</strong> &amp; Rothfels, C. J. (2021). <em>Mistreating birth-death models as priors in phylogenetic analysis compromises our ability to compare
models.</em> <u>bioRxiv</u>; submitted. <a href="https://www.biorxiv.org/content/10.1101/2021.07.12.452074v1">Link</a>. <a href="https://www.youtube.com/watch?v=SWldXeWet68&amp;feature=youtu.be">Slides</a>.</li>
  <li>Tribble, C. M., Freyman, W. A., Landis, M. J., Lim, J. Y., Barido-Sottani, J., Kopperud, B. T., Höhna, S. &amp; <strong>May, M. R.</strong> (2021). <em>RevGadgets: an R Package for visualizing Bayesian phylogenetic analyses from RevBayes.</em> <u>bioRxiv</u>; in review at <u>Methods in Ecology and Evolution</u>. <a href="https://www.biorxiv.org/content/10.1101/2021.05.10.443470v1">Link</a>.</li>
  <li>Höhna, S., Freyman, W. A., Nolen, Z., Huelsenbeck, J. P., <strong>May, M. R.</strong>, &amp; Moore, B. R. (2021). <em>A Bayesian approach for estimating branch-specific speciation and extinction rates.</em> <u>bioRxiv</u>; in revision. <a href="https://www.biorxiv.org/content/10.1101/2021.07.12.452074v1">Link</a>.</li>
  <li>Gao, J., <strong>May, M. R.</strong>, Rannala, B. &amp; Moore, B. R. (2021). <em>Phylodynamic insights on the early spread of the COVID-19 pandemic and the efficacy of intervention measures.</em> <u>bioRxiv</u>; in revision. <a href="https://www.biorxiv.org/content/10.1101/2021.05.01.442286v1">Link</a>.</li>
  <li>Barido-Sottani, J., Justison, J. A., Borges, Rui., Brown, J. M., Dismukes, W., do Rosario, P., ..., <strong>May, M. R.</strong>, ..., &amp; Heath, T. A. (2021). <em>Lessons learned from organizing and teaching virtual phylogenetics workshops.</em> In review at <u>Bulletin of the Society of Systematic Biologists</u>.</li>
  <li><strong>May, M. R.</strong>, Contreras, D. L., Sundue, M. A., Nagalingum, N. S., Looy, C. V., &amp; Rothfels, C. J. (2021). <em>Inferring the total-evidence timescale of Marattialean fern evolution in the face of model sensitivity.</em> <u>Systematic Biology</u>, in press. <a href="https://doi.org/10.1093/sysbio/syab020">Link</a>. <a href="https://www.youtube.com/watch?v=5H_u9bjn064">Slides</a>.</li>
  <li>Thompson A., <strong>May M. R.</strong>, Moore, B. R. &amp; Kopp A. (2020). <em>A hierarchical Bayesian mixture model for inferring the expression state of genes in transcriptomes.</em> <u>Proceedings of the National Academy of Sciences</u> 117(32), 19339–19346. <a href="https://doi.org/10.1073/pnas.1919748117">Link</a>.</li>
  <li><strong>May, M. R.</strong> &amp; Moore, B. R. (2020). <em>A Bayesian approach for inferring the impact of a discrete character on rates of continuous-character evolution in the presence of background-rate variation.</em> <u>Systematic Biology</u> 69(3), 530–544. <a href="https://doi.org/10.1093/sysbio/syz069">Link</a>.</li>
  <li>Turelli, M., Cooper, B. S., Richardson, K. M., Ginsberg, P. S., Peckenpaugh, B., Antelope, C. X., ..., <strong>May, M. R.</strong>, ... &amp; Hoffmann, A. A. (2018). <em>Rapid global spread of wRi-like Wolbachia across multiple Drosophila.</em> <u>Current Biology</u> 28(6), 963–971. <a href="https://linkinghub.elsevier.com/retrieve/pii/S0960982218301696">Link</a>.</li>
  <li>Jiao, Y., Peluso, P., Shi, J., Liang, T., Stitzer, M. C., Wang, B., ..., <strong>May, M. R.</strong> ,... &amp; Ware, D. (2017). <em>Improved maize reference genome with single-molecule technologies.</em> <u>Nature</u> 546(7659), 524–527. <a href="https://www.nature.com/articles/nature22971">Link</a>.</li>
  <li><strong>May, M. R.</strong>, S. Höhna, &amp; B. R. Moore (2016). <em>A Bayesian approach for detecting the impact of mass-extinction events on molecular phylogenies when rates of lineage diversification may vary.</em> <u>Methods in Ecology and Evolution</u> 7(8), 947–959. <a href="https://doi.org/10.1111/2041-210X.12563">Link</a>.</li>
  <li>Moore, B. R., Höhna, S., <strong>May, M. R.</strong>, Rannala, B. &amp; Huelsenbeck, J. P. (2016). <em>Critically evaluating the theory and performance of Bayesian analysis of macroevolutionary mixtures.</em> <u>Proceedings of the National Academy of Sciences</u> 34(113), 9569–9574. <a href="https://doi.org/10.1073/pnas.1518659113">Link</a>.</li>
  <li><strong>May, M. R.</strong> &amp; Moore, B. R. (2016). <em>How well can we detect lineage-specific diversification-rate shifts? A simulation study of sequential AIC methods.</em> <u>Systematic Biology</u> 6(65), 1076–1084. <a href="https://doi.org/10.1093/sysbio/syw026">Link</a>.</li>
  <li>Höhna, S., <strong>May, M. R.</strong>, &amp; Moore, B. R. (2016). <em>TESS: an R package for efficiently simulating phylogenetic trees and performing Bayesian inference of lineage diversification rates.</em> <u>Bioinformatics</u> 32(5), 789–791. <a href="https://doi.org/10.1093/bioinformatics/btv651">Link</a>.</li>
  <li>Magee, A. F., <strong>May, M. R.</strong>, &amp; Moore, B. R. (2014). <em>The dawn of open access to phylogenetic data.</em> <u>PLoS One</u> 9(10), e110268. <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0110268">Link</a>.</li>
  <li>Brandvain, Y., Pauly, G. B., <strong>May, M. R.</strong>, &amp; Turelli, M. (2014). <em>Explaining Darwin’s corollary to Haldane’s rule: the role of mitonuclear interactions in asymmetric postzygotic isolation among toads</em>. <u>Genetics</u> 197(2), 743–747. <a href="https://doi.org/10.1534/genetics.113.161133">Link</a>.</li>
  <li>Melters, D. P., Bradnam, K. R., Young, H. A., Telis, N., <strong>May, M. R.</strong>, Ruby, J. G., ... &amp; Chan, S. W. (2013). <em>Comparative analysis of tandem
repeats from hundreds of species reveals unique insights into centromere evolution.</em> <u>Genome Biology</u> 14(1), 1–20. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2013-14-1-r10">Link</a>.</li>
  <li><strong>May, M. R.</strong>, Provance, M. C., Sanders, A. C., Ellstrand, N. C., &amp; Ross-Ibarra, J. (2009). <em>A Pleistocene clone of Palmer’s oak persisting in southern California.</em> <u>PLoS One</u> 4(12), e8346. <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0008346">Link</a>.</li>
</ol>

<h2>Presentations</h2>

<ol reversed="reversed">
  <li><strong>May, M. R.</strong> &amp; Rothfels, C. J. (2021). <em>Mistreating birth-death models as priors in phylogenetic analysis compromises our ability to compare models.</em> <u>Botanical Society of America Conference</u>. (Invited.)</li>
  <li><strong>May, M. R.</strong>, Contreras, D. L., Sundue, M. A., Nagalingum, N. S., Looy, C. V., &amp; Rothfels, C. J. (2021). <em>Inferring the total-evidence timescale of marattialean fern evolution in the face of model sensitivity.</em> <u>Joint Annual Evolution Meeting</u>.</li>
  <li><strong>May, M. R.</strong> &amp; Moore, B. R. (2017). <em>A Bayesian method for inferring discrete-state dependent rates of continuous-trait evolution.</em> <u>Joint
Annual Evolution Meeting; Portland, Oregon</u>.</li>
  <li><strong>May, M. R.</strong>, Moore, B. R., Höhna, S., Huelsenbeck, J. P., &amp; Rannala, B. (2016). <em>Towards a correct Bayesian method for inferring lineage-specific rates of speciation and extinction.</em> <u>Joint Annual Evolution Meeting; Austin, Texas</u>.</li>
  <li><strong>May, M. R.</strong> (2016). <em>Problems and solutions for inferring lineage-specific rates of speciation and, extinction.</em> <u>Center for Population Biology seminar series; UC Davis</u>.</li>
  <li><strong>May, M. R.</strong> &amp; Moore, B. R. (2014). <em>Inferring lineage-specific rates of continuous-trait evolution.</em> <u>Botanical Society of America Conference; Boise, Idaho</u>.</li>
  <li><strong>May, M. R.</strong> (2013). <em>Mostly Bayesian methods for inferring heterogeneous evolutionary processes across phylogenies.</em> <u>Center for Population Biology seminar series; UC Davis</u>.</li>
  <li><strong>May, M. R.</strong> &amp; Moore, B. R., (2013). <em>Assessing the reliability of MEDUSA using simulation.</em> <u>Joint Annual Evolution Meeting; Snowbird, Utah</u>.</li>
</ol>

<h2>Fellowships and Awards</h2>

<p>UC Davis Outstanding Graduate Student Teaching Award (2017)<br />
NSF GRFP Predoctoral Fellowship (2011–2016)</p>

<h2>Societies and Services</h2>

<p><strong>Member of:</strong> <em>Society of Systematic Biologist</em>, <em>Society for the Study of Evolution</em>, <em>Botanical Society of America</em>.<br />
<br />
<strong>Peer review for:</strong> <em>Systematic Biology</em>, <em>Proceedings of the National Academy of Sciences</em>, <em>Proceedings of the Royal Society B</em>, <em>Methods in Ecology and Evolution</em>, <em>Frontiers in Plant Science</em>, <em>Functional Ecology</em>, <em>Molecular Biology and Evolution</em>, <em>PLoS One</em>, <em>PeerJ</em>.</p>

<h2>Teaching</h2>

<table>
  <tbody>
    <tr>
      <td>2021</td>
      <td>Instructor for Stay-At-Home RevBayes Workshop.</td>
    </tr>
    <tr>
      <td>2019</td>
      <td>Instructor for Madrid Phylogenetics Workshop.</td>
    </tr>
    <tr>
      <td>2019</td>
      <td>Organizer and instructor for Bodega Applied Phylogenetics Workshop.</td>
    </tr>
    <tr>
      <td>2018</td>
      <td>Instructor for Madrid Phylogenetics Workshop.</td>
    </tr>
    <tr>
      <td>2018</td>
      <td>Teaching assistant for EVE 103: Phylogeny and Macroevolution (UC Davis).</td>
    </tr>
    <tr>
      <td>2017</td>
      <td>Organizer and instructor for Bodega Applied Phylogenetics Workshop.</td>
    </tr>
    <tr>
      <td>2016</td>
      <td>Teaching assistant for EVE 103: Phylogeny and Macroevolution (UC Davis). <em>Awarded UC Davis Outstanding Graduate Student Teaching Award 2017.</em></td>
    </tr>
    <tr>
      <td>2015</td>
      <td>Organizer and teaching assistant for Bodega Applied Phylogenetics Workshop.</td>
    </tr>
    <tr>
      <td>2014</td>
      <td>Organizer and teaching assistant for Bodega Applied Phylogenetics Workshop.</td>
    </tr>
    <tr>
      <td>2013</td>
      <td>Organizer and teaching assistant for Bodega Applied Phylogenetics Workshop.</td>
    </tr>
    <tr>
      <td>2012</td>
      <td>Teaching assistant for EVE 103: Phylogeny and Macroevolution (UC Davis).</td>
    </tr>
  </tbody>
</table>

<h2>Software Tutorials</h2>

<ol reversed="reversed">
  <li><strong>May, M. R.</strong> &amp; Höhna, S. <em>Rev Language Syntax: A very short introduction to the Rev language.</em> <a href="https://revbayes.github.io/tutorials/intro/rev.html">Link</a>.</li>
  <li>Tribble, C. M. &amp; <strong>May, M. R.</strong> <em>Introduction to RevGadgets: Plotting the output of RevBayes analyses in the R package RevGadgets.</em> <a href="https://revbayes.github.io/tutorials/intro/revgadgets.html">Link</a>.</li>
  <li><strong>May, M. R.</strong>, Moore, B. R., &amp; Höhna, S. <em>Introduction to MCMC using RevBayes: Introduction to MCMC simulation using a simple binomial model.</em> <a href="https://revbayes.github.io/tutorials/mcmc/binomial.html">Link</a>.</li>
  <li><strong>May, M. R.</strong> &amp; Höhna, S. <em>Model Selection of Partition Models: Comparing relative model fit with Bayes factors.</em> <a href="https://revbayes.github.io/tutorials/model_selection_bayes_factors/bf_partition_model.html">Link</a>.</li>
  <li><strong>May, M. R.</strong> &amp; Höhna, S. <em>Model Averaging of Substitution Models: Reversible-jump MCMC over substitution models.</em> <a href="https://revbayes.github.io/tutorials/model_selection_bayes_factors/bf_subst_model_rj.html">Link</a>.</li>
  <li>Höhna, S. &amp; <strong>May, M. R.</strong> <em>Branch-Specific Diversification Rate Estimation: How to estimate branch-specific shifts in diversification rates.</em> <a href="https://revbayes.github.io/tutorials/divrate/branch_specific.html">Link</a>.</li>
  <li>Höhna, S., Freyman, W. A., &amp; <strong>May, M. R.</strong> <em>Diversification Rate Estimation with Missing Taxa: How to estimate diversification rates with incomplete taxon sampling.</em> <a href="https://revbayes.github.io/tutorials/divrate/sampling.html">Link</a></li>
  <li><strong>May, M. R.</strong> <em>Simple Brownian Rate Estimation: Estimating rates of Brownian-motion evolution.</em> <a href="https://revbayes.github.io/tutorials/cont_traits/simple_bm.html">Link</a>.</li>
  <li><strong>May, M. R.</strong> <em>Relaxed Brownian Rate Estimation: Estimating branch-specific rates of Brownian-motion evolution.</em> <a href="https://revbayes.github.io/tutorials/cont_traits/relaxed_bm.html">Link</a>.</li>
  <li><strong>May, M. R.</strong> <em>State-Dependent Brownian Rate Estimation: Estimating state-dependent rates of Brownian-motion evolution.</em> <a href="https://revbayes.github.io/tutorials/cont_traits/state_dependent_bm.html">Link</a>.</li>
  <li><strong>May, M. R.</strong> <em>Multivariate Brownian Motion: Accounting for correlations among continuous traits.</em> <a href="https://revbayes.github.io/tutorials/cont_traits/multivariate_bm.html">Link</a>.</li>
  <li><strong>May, M. R.</strong> <em>Simple Ornstein-Uhlenbeck Models: Estimating optima under Ornstein-Uhlenbeck evolution.</em> <a href="https://revbayes.github.io/tutorials/cont_traits/simple_ou.html">Link</a>.</li>
  <li><strong>May, M. R.</strong> <em>Relaxed Ornstein-Uhlenbeck Models: Estimating lineage-specific optima under Ornstein-Uhlenbeck evolution.</em> <a href="https://revbayes.github.io/tutorials/cont_traits/relaxed_ou.html">Link</a>.</li>
</ol>
