Center for Population Biology, University of California, Davis  
Email: [`mikeryanmay@gmail.com`](mailto:mikeryanmay@gmail.com)  
Website: [`mikeryanmay.github.io`](https://mikeryanmay.github.io)  

## Education and Appointments

| 2020--present | Postdoctoral scholar, UC Davis |
| 2019--2021    | Postdoctoral scholar, UC Berkeley |
| 2011--2019    | PhD, Population Biology, UC Davis |
| 2008--2010    | BS, Evolution, Ecology and Biodiversity, UC Davis (Highest Honors) |

## Publications

21. **May, M. R.** & Rothfels, C. J. (2022). *Diversification models conflate likelihood and prior, and cannot be compared using conventional model-comparison tools.* `bioRxiv`; submitted. [Link](https://www.biorxiv.org/content/10.1101/2021.07.12.452074v1). [Slides](https://www.youtube.com/watch?v=SWldXeWet68&feature=youtu.be).
20. Tribble, C. M., **May, M. R.**, Jackson-Gain, A., Zenil-Ferguson, R., Specht, C. D., & Rothfels, C. J. (2022) *Unearthing modes of climatic adaptation in underground storage organs across Liliales.* `bioRxiv`; submitted. [Link](https://doi.org/10.1101/2021.09.03.458928)
19. Höhna, S., Freyman, W. A., Nolen, Z., Huelsenbeck, J. P., **May, M. R.**, & Moore, B. R. (2022). *A Bayesian approach for estimating branch-specific speciation and extinction rates.* `bioRxiv`; in revision. [Link](https://www.biorxiv.org/content/10.1101/2021.07.12.452074v1).
18. Gao, J., **May, M. R.**, Rannala, B. & Moore, B. R. (2022). *PrioriTree: a utility for improving phylodynamic analyses in BEAST.* `bioRxiv`; submitted. [Link](doi: https://doi.org/10.1101/2022.08.24.505196).
17. Gao, J., **May, M. R.**, Rannala, B. & Moore, B. R. (2022). *Model misspecification misleads inference of the spatial dynamics of disease outbreaks.* `medRxiv`; submitted. [Link](https://doi.org/10.1101/2022.08.24.22278802).
16. Gao, J., **May, M. R.**, Rannala, B. & Moore, B. R. (2022). *New phylogenetic models incorporating interval-specific dispersal dynamics improve inference of disease spread.* `Molecular Biology and Evolution`. [Link](https://doi.org/10.1093/molbev/msac159).
15. Barido-Sottani, J., Justison, J. A., Borges, Rui., Brown, J. M., Dismukes, W., do Rosario, P., \..., **May, M. R.**, \..., & Heath, T. A. (2022). *Lessons learned from organizing and teaching virtual phylogenetics workshops.* `Bulletin of the Society of Systematic Biologists`. [Link](https://doi.org/10.18061/bssb.v1i2.8425)
14. Tribble, C. M., Freyman, W. A., Landis, M. J., Lim, J. Y., Barido-Sottani, J., Kopperud, B. T., Höhna, S. & **May, M. R.** (2022). *RevGadgets: an R Package for visualizing Bayesian phylogenetic analyses from RevBayes.* `Methods in Ecology and Evolution`. [Link](https://doi.org/10.1111/2041-210X.13750).
13. **May, M. R.**, Contreras, D. L., Sundue, M. A., Nagalingum, N. S., Looy, C. V., & Rothfels, C. J. (2021). *Inferring the total-evidence timescale of Marattialean fern evolution in the face of model sensitivity.* `Systematic Biology`. [Link](https://doi.org/10.1093/sysbio/syab020). [Slides](https://www.youtube.com/watch?v=5H_u9bjn064).
12. Thompson A., **May M. R.**, Moore, B. R. & Kopp A. (2020). *A hierarchical Bayesian mixture model for inferring the expression state of genes in transcriptomes.* `Proceedings of the National Academy of Sciences` 117(32), 19339--19346. [Link](https://doi.org/10.1073/pnas.1919748117).
11. **May, M. R.** & Moore, B. R. (2020). *A Bayesian approach for inferring the impact of a discrete character on rates of continuous-character evolution in the presence of background-rate variation.* `Systematic Biology` 69(3), 530--544. [Link](https://doi.org/10.1093/sysbio/syz069).
10. Turelli, M., Cooper, B. S., Richardson, K. M., Ginsberg, P. S., Peckenpaugh, B., Antelope, C. X., \..., **May, M. R.**, \... & Hoffmann, A. A. (2018). *Rapid global spread of wRi-like Wolbachia across multiple Drosophila.* `Current Biology` 28(6), 963--971. [Link](https://linkinghub.elsevier.com/retrieve/pii/S0960982218301696).
9. Jiao, Y., Peluso, P., Shi, J., Liang, T., Stitzer, M. C., Wang, B., \..., **May, M. R.** ,\... & Ware, D. (2017). *Improved maize reference genome with single-molecule technologies.* `Nature` 546(7659), 524--527. [Link](https://www.nature.com/articles/nature22971).
8. **May, M. R.**, S. Höhna, & B. R. Moore (2016). *A Bayesian approach for detecting the impact of mass-extinction events on molecular phylogenies when rates of lineage diversification may vary.* `Methods in Ecology and Evolution` 7(8), 947--959. [Link](https://doi.org/10.1111/2041-210X.12563).
7. Moore, B. R., Höhna, S., **May, M. R.**, Rannala, B. & Huelsenbeck, J. P. (2016). *Critically evaluating the theory and performance of Bayesian analysis of macroevolutionary mixtures.* `Proceedings of the National Academy of Sciences` 34(113), 9569--9574. [Link](https://doi.org/10.1073/pnas.1518659113).
6. **May, M. R.** & Moore, B. R. (2016). *How well can we detect lineage-specific diversification-rate shifts? A simulation study of sequential AIC methods.* `Systematic Biology` 6(65), 1076--1084. [Link](https://doi.org/10.1093/sysbio/syw026).
5. Höhna, S., **May, M. R.**, & Moore, B. R. (2016). *TESS: an R package for efficiently simulating phylogenetic trees and performing Bayesian inference of lineage diversification rates.* `Bioinformatics` 32(5), 789--791. [Link](https://doi.org/10.1093/bioinformatics/btv651).
4. Magee, A. F., **May, M. R.**, & Moore, B. R. (2014). *The dawn of open access to phylogenetic data.* `PLoS One` 9(10), e110268. [Link](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0110268).
3. Brandvain, Y., Pauly, G. B., **May, M. R.**, & Turelli, M. (2014). *Explaining Darwin's corollary to Haldane's rule: the role of mitonuclear interactions in asymmetric postzygotic isolation among toads*. `Genetics` 197(2), 743--747. [Link](https://doi.org/10.1534/genetics.113.161133).
2. Melters, D. P., Bradnam, K. R., Young, H. A., Telis, N., **May, M. R.**, Ruby, J. G., \... & Chan, S. W. (2013). *Comparative analysis of tandem
repeats from hundreds of species reveals unique insights into centromere evolution.* `Genome Biology` 14(1), 1--20. [Link](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2013-14-1-r10).
1. **May, M. R.**, Provance, M. C., Sanders, A. C., Ellstrand, N. C., & Ross-Ibarra, J. (2009). *A Pleistocene clone of Palmer's oak persisting in southern California.* `PLoS One` 4(12), e8346. [Link](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0008346).
{: reversed="reversed"}

## Presentations

9. **May, M. R.** & Meyer, X. (2022). *TensorPhylo: a high-performance library for a general class of state-dependent birth-death models* `Joint Annual Evolution Meeting`.
8. **May, M. R.** & Rothfels, C. J. (2021). *Mistreating birth-death models as priors in phylogenetic analysis compromises our ability to compare models.* `Botanical Society of America Conference`. (Invited.)
7. **May, M. R.**, Contreras, D. L., Sundue, M. A., Nagalingum, N. S., Looy, C. V., & Rothfels, C. J. (2021). *Inferring the total-evidence timescale of marattialean fern evolution in the face of model sensitivity.* `Joint Annual Evolution Meeting`.
6. **May, M. R.** & Moore, B. R. (2017). *A Bayesian method for inferring discrete-state dependent rates of continuous-trait evolution.* `Joint
Annual Evolution Meeting; Portland, Oregon`.
5. **May, M. R.**, Moore, B. R., Höhna, S., Huelsenbeck, J. P., & Rannala, B. (2016). *Towards a correct Bayesian method for inferring lineage-specific rates of speciation and extinction.* `Joint Annual Evolution Meeting; Austin, Texas`.
4. **May, M. R.** (2016). *Problems and solutions for inferring lineage-specific rates of speciation and, extinction.* `Center for Population Biology seminar series; UC Davis`.
3. **May, M. R.** & Moore, B. R. (2014). *Inferring lineage-specific rates of continuous-trait evolution.* `Botanical Society of America Conference; Boise, Idaho`.
2. **May, M. R.** (2013). *Mostly Bayesian methods for inferring heterogeneous evolutionary processes across phylogenies.* `Center for Population Biology seminar series; UC Davis`.
1. **May, M. R.** & Moore, B. R., (2013). *Assessing the reliability of MEDUSA using simulation.* `Joint Annual Evolution Meeting; Snowbird, Utah`.
{: reversed="reversed"}

## Fellowships and Awards

UC Davis Outstanding Graduate Student Teaching Award (2017)  
NSF GRFP Predoctoral Fellowship (2011--2016)

## Societies and Services

**Member of:** _Society of Systematic Biologist_, _Society for the Study of Evolution_, _Botanical Society of America_.  
<br/>
**Peer review for:** _Systematic Biology_, _Proceedings of the National Academy of Sciences_, _Proceedings of the Royal Society B_, _Methods in Ecology and Evolution_, _Frontiers in Plant Science_, _Functional Ecology_, _Molecular Biology and Evolution_, _PLoS One_, _PeerJ_.

## Teaching

| 2022 | Instructor for Computational Molecular Evolution (CoME) Workshop (Hinxton, UK). |
| 2022 | Organizer for Divergence-time Estimation Workshop. [Link](https://mikeryanmay.github.io/ted.workshop.github.io/). |
| 2021 | Instructor for Stay-At-Home RevBayes Workshop. |
| 2019 | Instructor for Madrid Phylogenetics Workshop. |
| 2019 | Organizer and instructor for Bodega Applied Phylogenetics Workshop. |
| 2018 | Instructor for Madrid Phylogenetics Workshop. |
| 2018 | Teaching assistant for EVE 103: Phylogeny and Macroevolution (UC Davis). |
| 2017 | Organizer and instructor for Bodega Applied Phylogenetics Workshop. |
| 2016 | Teaching assistant for EVE 103: Phylogeny and Macroevolution (UC Davis). _Awarded UC Davis Outstanding Graduate Student Teaching Award 2017._ |
| 2015 | Organizer and teaching assistant for Bodega Applied Phylogenetics Workshop. |
| 2014 | Organizer and teaching assistant for Bodega Applied Phylogenetics Workshop. |
| 2013 | Organizer and teaching assistant for Bodega Applied Phylogenetics Workshop. |
| 2012 | Teaching assistant for EVE 103: Phylogeny and Macroevolution (UC Davis). |

## Software Tutorials

14. **May, M. R.** *Total-evidence dating, model sensitivity, and model comparison.* [Link](https://revbayes.github.io/tutorials/ted_workflow/)
13. **May, M. R.** & Höhna, S. *Rev Language Syntax: A very short introduction to the Rev language.* [Link](https://revbayes.github.io/tutorials/intro/rev.html).
12. Tribble, C. M. & **May, M. R.** *Introduction to RevGadgets: Plotting the output of RevBayes analyses in the R package RevGadgets.* [Link](https://revbayes.github.io/tutorials/intro/revgadgets.html).
11. **May, M. R.**, Moore, B. R., & Höhna, S. *Introduction to MCMC using RevBayes: Introduction to MCMC simulation using a simple binomial model.* [Link](https://revbayes.github.io/tutorials/mcmc/binomial.html).
10. **May, M. R.** & Höhna, S. *Model Selection of Partition Models: Comparing relative model fit with Bayes factors.* [Link](https://revbayes.github.io/tutorials/model_selection_bayes_factors/bf_partition_model.html).
9. **May, M. R.** & Höhna, S. *Model Averaging of Substitution Models: Reversible-jump MCMC over substitution models.* [Link](https://revbayes.github.io/tutorials/model_selection_bayes_factors/bf_subst_model_rj.html).
8. Höhna, S. & **May, M. R.** *Branch-Specific Diversification Rate Estimation: How to estimate branch-specific shifts in diversification rates.* [Link](https://revbayes.github.io/tutorials/divrate/branch_specific.html).
7. Höhna, S., Freyman, W. A., & **May, M. R.** *Diversification Rate Estimation with Missing Taxa: How to estimate diversification rates with incomplete taxon sampling.* [Link](https://revbayes.github.io/tutorials/divrate/sampling.html)
6. **May, M. R.** *Simple Brownian Rate Estimation: Estimating rates of Brownian-motion evolution.* [Link](https://revbayes.github.io/tutorials/cont_traits/simple_bm.html).
5. **May, M. R.** *Relaxed Brownian Rate Estimation: Estimating branch-specific rates of Brownian-motion evolution.* [Link](https://revbayes.github.io/tutorials/cont_traits/relaxed_bm.html).
4. **May, M. R.** *State-Dependent Brownian Rate Estimation: Estimating state-dependent rates of Brownian-motion evolution.* [Link](https://revbayes.github.io/tutorials/cont_traits/state_dependent_bm.html).
3. **May, M. R.** *Multivariate Brownian Motion: Accounting for correlations among continuous traits.* [Link](https://revbayes.github.io/tutorials/cont_traits/multivariate_bm.html).
2. **May, M. R.** *Simple Ornstein-Uhlenbeck Models: Estimating optima under Ornstein-Uhlenbeck evolution.* [Link](https://revbayes.github.io/tutorials/cont_traits/simple_ou.html).
1. **May, M. R.** *Relaxed Ornstein-Uhlenbeck Models: Estimating lineage-specific optima under Ornstein-Uhlenbeck evolution.* [Link](https://revbayes.github.io/tutorials/cont_traits/relaxed_ou.html).
{: reversed="reversed"}
